core/core.mk

```diff
@@ -49,6 +49,9 @@ endif
 ifneq ($(CFG_TEE_CORE_DEBUG),y)
 cppflags$(sm)  += -DNDEBUG
 endif
+ifeq ($(CFG_BGET_BESTFIT),y)
```

Comment:
The name of the flag should reflect that it applies to the TEE core only (i.e., not to the `malloc()` supplied to TAs). `CFG_CORE_BGET_BESTFIT` maybe?